server/src/main/java/org/kaazing/gateway/server/GatewayCLI.java

```diff
@@ -32,20 +32,18 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Main entry point for gateway process (at least when being called by the start scripts--it's possible to call Gateway directly
- * from other Java code as well.) This generic processor is called from all three Mains.
+ * Main entry point for gateway process when started by the command line.
  * <p/>
- * As of 3.2 we support two arguments to main: --config <configFile> --help (we also support the form /config <configFile>) The
- * value of <configFile> is considered as a path to the gateway configuration file--NOT just a file name. If provided by the
- * caller, the property is converted to a system property and passed to Gateway, rather than requiring Gateway to handle a new
- * input vector.
+ * Supported command line arguments are --config <configFile> and --help. The value of <configFile> is considered as a
+ * path to the gateway configuration file--NOT just a file name. If provided by the caller, the property is converted to
+ * a system property and passed to Gateway, rather than requiring Gateway to handle a new input vector.
  */
-public class GatewayCommandProcessor {
-    private static final Logger LOGGER = LoggerFactory.getLogger(GatewayCommandProcessor.class);
+public class GatewayCLI {
```

Comment:
Could this be GatewayCommandLineProcessor or something similar that isn't an abbreviation?  Everything else looks good.